assets/panel/prefs.js

```diff
@@ -50,7 +50,7 @@ pref("devtools.debugger.skip-pausing", false);
 pref("devtools.debugger.features.wasm", true);
 pref("devtools.debugger.features.shortcuts", true);
 pref("devtools.debugger.features.root", true);
-pref("devtools.debugger.features.column-breakpoints", false);
+pref("devtools.debugger.features.column-breakpoints", true);
```

Comment:
👍

Reply:
Actually, as things stand, we'll want to change this back to `false` before merging.  I just pressed `option` in a minified file and my browser hung, so we'll need to measure performance implications here.  Keep it true for now, if only for the sake of everyone testing and tests running.